glean-core/src/metrics/time_unit.rs

```diff
@@ -11,12 +11,19 @@ use crate::error::{Error, ErrorKind};
 /// metric types (e.g. DatetimeMetric).
 #[derive(Copy, Clone, Debug, Deserialize, Serialize)]
 pub enum TimeUnit {
+    /// Format to nanosecond precision.
```

Comment:
I would say "truncate to".  It both (a) expresses that we use truncation and not rounding, (b) is generic between DateTime (where format makes sense) and Timespan (where it doesn't).